help/destinations/catalog/social/tiktok.md

```diff
@@ -0,0 +1,114 @@
+---
+title: TikTok connection // Replace with the name of destination
```

Comment:
Please make sure to delete the `// Replace with the name of destination` part from the metadata